src/kudu/consensus/raft_consensus.cc

```diff
@@ -150,7 +155,23 @@ DEFINE_bool(raft_attempt_to_replace_replica_without_majority, false,
             "Warning! This is only intended for testing.");
 TAG_FLAG(raft_attempt_to_replace_replica_without_majority, unsafe);
 
+DEFINE_bool(raft_enable_multi_hop_proxy_routing, true,
```

Comment:
maybe set this by default false,  and then turn it  on in fbcode?

Reply:
Sure, will do.